api_v3/services/BaseEntryService.php

```diff
@@ -560,6 +603,27 @@ function listAction(KalturaBaseEntryFilter $filter = null, KalturaFilterPager $p
 		$response = new KalturaBaseEntryListResponse();
 		$response->objects = $result->objects;
 		$response->totalCount = $result->totalCount;
+		
+		$defaultLanguage = $this->getPartner()->getDefaultLanguage();
```

Comment:
the multilang logic is not related to a specific api, we need to return the correct result for any api that returns the list, so this logic also need to move to inner function